docs/wfc3tools/wf32d.rst

```diff
@@ -20,8 +21,20 @@ executed, after which the switch will be set to COMPLETE in the corresponding
 output files. See `Section 3.4.3 of the WFC3 Data Handbook <https://hst-docs.stsci.edu/wfc3dhb>`_ for more information.
 
 
+Running `wf32d` from a Python Terminal
+=======================================
+
+.. code-block:: shell
+
+    from wfc3tools import wf32d
+    wf32d(filename)
+
+
```

Comment:
No section needed for Displaying output from wf32d in a Jupyter Notebook?

Reply:
Duplicated text as seemingly done for the other sub-components.  Addessed in commit f4306cb.